json-lint/schemas/inventory/adservers/adsense.json

```diff
@@ -61,16 +61,32 @@
       },
       "auto-format": { "const" : "rspv" },
       "width": {
-        "type": ["integer","string"],
-        "title": "String of digits ( if not set, default is 300 )",
-        "description": "example: 300",
-        "pattern": "^(\\$\\{.+\\})$"  
+        "oneOf": [
+          {
+            "type": "integer"
+          },
+          {
+            "type": "string",
+            "title": "String of digits ( if not set, default is 300 )",
+            "description": "example: 300",
+            "pattern": "^\\$\\{.+\\}$"
+          },
+          {
+            "type": "string",
+            "title": "window width",
+            "description": "example: 300vw",
+            "pattern": "^[0-9]+vw$"
```

Comment:
lets add in dimensions.json the const option of 100vw.
You can use the oneOf, is more clear the split options

Reply:
Fixed